netlify/functions/deploy-succeeded.js

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line import/no-unresolved
 const fetch = require('node-fetch');
```

Comment:
Is this even needed still, since we started building in Node.js 18? It should have a global `fetch` available now.

Reply:
Thanks for spotting it, using native Node `fetch` works without issues: https://app.netlify.com/sites/material-ui-x/logs/functions?scope=deploypreview:14105 👍